Citizen Science API.R

```diff
@@ -0,0 +1,172 @@
+library(rgbif)
```

Comment:
It appears that you changed the file name to switch _ to spaces. Please change back. First, it's bad practice to include spaces in file names. Second, the change in name makes it impossible to see the difference between the previous version and the new version -- git things these are completely different files.